DIPs/DIP-15.md

```diff
@@ -0,0 +1,62 @@
+---
+DIP: 15
+Title: Travel insurance
+Status: Draft
+Themes: Travel, Safety
+Tags: DeFi, Insurance, Peer2Peer
+Authors: christoph@etherisc.com
+Resources Required: CustomerSupportBooth at DevConVI
+Discussion: https://forum.devcon.org/t/dip-15-travel-insurance/176
+Created: 2020-11-27
+---
+
+## Summary of Proposal
+
+A lot of people will travel to DevConVI. This will impose a number of risks to these people:
+flight Delays, baggage loss, theft, and others. We will build a decentralized travel insurance for DevConVI participants. 
+
+## Abstract
+We will offer a FlightDelay Insurance for DevconVI attendees. 
+
+After registering, people select their flight(s) and pay a small premium (about USD 10$-25$) in some stablecoin,
+e.g. USDC, DAI etc. 
+Via an oracle, the risk of the flight being late is calculated. 
+After landing, the actual delay is determined by a second oracle call and in case of a delay, the payout is calcuated
+and immediately transfered to the customers wallet.
+We are currently talking to [Team WindingTree](https://windingtree.com) to integrate the insurance in windingtree's travel agency solution.
+The oracle is provided by [Chainlink](https://chain.link). The core business process runs on our 
+"[Generic Insurance Framework](https://github.com/etherisc/GIF)" and does
+not necessarily need any off-chain components, which are, however, provided for convenience. 
+
+We will explore if other travel related insurance products can be implemented in a similar way, e.g. baggage loss,
+theft, and others, depending on availability of reliable loss data. 
+
+Claims are paid out from risk pools. Investors can provide liquidity for risk pools. Liquidity providers receive
+those premiums which are not used to pay out claims. To become a liquidity provider, investors need to 
+stake DIP tokens.
```

Comment:
wonder why they are called DIP tokens

Reply:
Buy the f*ckin DIP ;-)
No honestly: Decentralized Insurance Protocol (... Token)

Reply:
ah I see - maybe add this in brackets behind - others might also confuse it with Devcon Improvement Proposal tokens (that do not exist) like me - especially in the scope of this repo ;-)

Reply:
Fixed